datasets/us/fincen_special_measures/crawler.py

```diff
@@ -0,0 +1,147 @@
+from lxml import html
+import re
+from zavod import Context, helpers as h
+from normality import slugify
+from typing import Dict, Generator
+
+BASE_URL = "https://www.fincen.gov"
+
+
+def convert_date(date_str: str) -> str or None:
+    """Convert various date formats to 'YYYY-MM-DD'."""
+    # Regular expression to find dates in the format 'MM/DD/YYYY'
+    date_pattern = re.compile(r"(\d{1,2}/\d{1,2}/\d{4})")
+    match = date_pattern.search(date_str)
+    if match:
+        date_str = match.group(1)  # Extract the first matching date
```

Comment:
can you easily enough extract and return all matching dates? we support multiple values for the same property for just this scenario.

Reply:
maybe re.findall would do the trick here?